spec/request/flipper_request_spec.rb

```diff
@@ -3,25 +3,52 @@
 require 'rails_helper'
 
 RSpec.describe 'Flipper UI', type: :request do
+  before(:all) do
+    Rails.application.routes.draw do
+      mount Flipper::UI.app(
+        Flipper.instance,
+        rack_protection: { except: :authenticity_token }
```

Comment:
The addition of [this line](https://github.com/jnunemaker/flipper/blob/fa78a0030c7f139aecc3f9c8468baf9fd1498eb9/lib/flipper/ui.rb#L44) in the latest version makes setting this option possible.

Reply:
That's a good find!  

I'm a bit torn over whether we should override the route here or just change it in routes.rb.  Mostly, because if someone were to remove the `constraints` from routes tests would still pass.

Reply:
Refactored and added new spec that won't pass if constraint is not present.